core/src/main/java/nl/nn/adapterframework/senders/SenderSeries.java

```diff
@@ -105,19 +101,13 @@ public void iterateOverStatistics(StatisticsKeeperIterationHandler hski, Object
 		//hski.closeGroup(senderData);
 	}
 
-	@Override
-	public boolean isSynchronous() {
-		return synchronous;
-	}
-	public void setSynchronous(boolean value) {
-		synchronous = value;
-	}
 
 	@Override
 	@Deprecated // replaced by registerSender, to allow for multiple senders in XSD. 
 	public final void setSender(ISender sender) {
 		registerSender(sender);
 	}
```

Comment:
en kan je setSender niet gewoon weghalen? Ik zie niet waarom die setter daar nog is.. hij kan ook makkelijk uit de super classe.